dist/privacy-sandbox.js

```diff
@@ -0,0 +1,91 @@
+//[privacy-sandbox]
+// Topics API Usage Reference: https://developers.google.com/privacy-sandbox/relevance/topics/demo#the-topics-api-demo
+// Header Usage Reference: https://developers.google.com/privacy-sandbox/relevance/topics/demo#the-topics-api-demo
+
+let requests = $WPT_BODIES;
+const testURL = new URL($WPT_URL);
+let firstPartyDomain = testURL.hostname;
+
+result = {
+  topicsAvailable: document.featurePolicy.allowsFeature('browsing-topics'),
+  thirdPartiesUsingBrowsingTopics: {}
+}
+
+function fetchAndCheckAttestation(url) {
+  const controller = new AbortController();
+  const { signal } = controller;
+  setTimeout(() => controller.abort(), 5000);
+
+  try {
+    const response = fetch(url, { signal });
+    if (response.ok && response.headers.get('Content-Type').includes('application/json')) {
+      return true;
+    }
+    else {
+      return false;
```

Comment:
```suggestion
    return response ? true : false;
```
In no-cors request no properties are accessible via js.